yaml/com/ibm/PLDM/PCIeTopology.interface.yaml

```diff
@@ -0,0 +1,15 @@
+description: >
+      This interface is used by applications on bmc to request
+      the PCIE topology information.
+properties:
+     - name: PCIeTopologyRefresh
+       type: boolean
+       description: >
+         This property when set to 'true' refreshes the
+         topology information.
```

Comment:
After a refresh occurs this property should be set to 'false' ?

Reply:
Yes, when a refresh occurs this is set back to false.